src/N7consulting/PrivacyBundle/Resources/views/Privacy/access.html.twig

```diff
@@ -0,0 +1,69 @@
+{% extends "N7consultingPrivacyBundle::layout.html.twig" %}
+
+{% block content_title %}
+    {{ 'dashboard.vie_privee'|trans({}, 'dashboard') }} - {{ personne.nomFormel }}
+{% endblock %}
+
+{% block content_bundle %}
+    {# hack: use the json serialize result to display a list #}
```

Comment:
it is a big hack, why doing that? Also if you are serializing entities I think you're better of using the Serializer component rather than bloating the entities with JSON serialization which doesn't properly handle recursions

Reply:
I announced that GDPR would be implemented by the 18/02/2018 on Jeyser. I lack of time to manage it and tried to reduce dev time with such hack.

Anyway I updated it, template is now clean and exporter relies on Symfony serializer, thanks.